### PR TITLE
Feature(Authentication):RR-0018:Setup Login/Register flow

### PR DESCRIPTION
**Pull request by KagouSurGit**
https://trello.com/c/3InzUpcg/20-featrr-0018-setup-login-register-flow

### CODE REVIEW

**Review by v1ljar — DISMISSED** [review]

File: radicals/forms.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from django import forms
```
Comment (EimantasBlazevicius) [inline radicals/forms.py]: All the authentication needs to be done in the "front_office/accounts_page" app

**Review by EimantasBlazevicius — DISMISSED** [review]

**Comment by T6nisValk** [comment]
Navigation is created separately by me and included into the base, will finish it later today (14.12). I'm guessing you added it for testing purposes to the base file. - Shouldn't merge with current base.html changes?

I am not quite sure about this approach on authentication. Needs to have a separate app for it? I might have missed some class about dealing with forms.py?

**Review by Jann337 — DISMISSED** [review]

File: radicals/forms.py (hunk 1 above)
Comment (EimantasBlazevicius) [reply radicals/forms.py]: Move this file to the app where it will be used.

File: radicals/urls.py [hunk 2]
```diff
@@ -1,21 +0,0 @@
-"""radicals URL Configuration
```
Comment (EimantasBlazevicius) [inline radicals/urls.py]: radicals folder should have this urls.py file, your app should probably not even start without it.

File: templates/base.html [hunk 3]
```diff
@@ -13,6 +13,7 @@
   <body>
     {% block body %}{% endblock body %}
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-C6RzsynM9kWDrMNeT87bh95OGNyZPhcTNXj1NW7RuBCsyN/o0jlpcV8Qyq46cDfL" crossorigin="anonymous"></script>
+
```
Comment (EimantasBlazevicius) [inline templates/base.html]: remove empty line

**Review by EimantasBlazevicius — CHANGES_REQUESTED** [review]

**Review by Jann337 — APPROVED** [review]

**Review by EimantasBlazevicius — APPROVED** [review]

**Pull request merged**